presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -92,6 +92,10 @@ public void testDecimal()
         try (ConnectedStatement connectedStatement = newStatement()) {
             checkRepresentation(connectedStatement.getStatement(), "0.1", Types.DECIMAL, new BigDecimal("0.1"));
             checkRepresentation(connectedStatement.getStatement(), "DECIMAL '0.1'", Types.DECIMAL, new BigDecimal("0.1"));
```

Comment:
add tests for getLong

also, add a test for a value that cannot be represented exact as a double:

```
long outsideOfDoubleExactRange = 9223372036854775774L;
//noinspection ConstantConditions
verify((long) (double) outsideOfDoubleExactRange - outsideOfDoubleExactRange == 33, "outsideOfDoubleExactRange should not be exact-representable as a double");
checkRepresentation(connectedStatement.getStatement(), format("DECIMAL '%s'", outsideOfDoubleExactRange), Types.DECIMAL,
        (rs, column) -> {
            assertEquals(rs.getObject(column), new BigDecimal("9223372036854775774"));
            assertEquals(rs.getBigDecimal(column), new BigDecimal("9223372036854775774"));

            assertEquals(rs.getLong(column), 9223372036854775774L);

            assertEquals(rs.getDouble(column), 9.223372036854776E18);

            assertEquals(rs.getString(column), "9223372036854775774");
        });
```

Reply:
Added this to the unit tests.